[CI] MlTrainedModelsUpgradeIT testTrainedModelInference failing

**Build scan:**
https://gradle-enterprise.elastic.co/s/l3o4xprvhhchq/tests/:x-pack:qa:rolling-upgrade:v8.7.0%23oldClusterTest/org.elasticsearch.upgrades.MlTrainedModelsUpgradeIT/testTrainedModelInference

**Reproduction line:**
```
./gradlew ':x-pack:qa:rolling-upgrade:v8.7.0#oldClusterTest' -Dtests.class="org.elasticsearch.upgrades.MlTrainedModelsUpgradeIT" -Dtests.method="testTrainedModelInference" -Dtests.seed=E1C744BA9BC7DC7D -Dtests.bwc=true -Dtests.locale=es-PA -Dtests.timezone=Europe/Dublin -Druntime.java=17
```

**Applicable branches:**
main

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.upgrades.MlTrainedModelsUpgradeIT&tests.test=testTrainedModelInference

**Failure excerpt:**
```
java.lang.AssertionError: 
Expected: "stopped"
     but: was "started"

  at __randomizedtesting.SeedInfo.seed([E1C744BA9BC7DC7D:7E4B0A1C4A4EBFBE]:0)
  at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
  at org.junit.Assert.assertThat(Assert.java:956)
  at org.junit.Assert.assertThat(Assert.java:923)
  at org.elasticsearch.upgrades.MlTrainedModelsUpgradeIT.lambda$putAndStartDFAAndWaitForFinish$3(MlTrainedModelsUpgradeIT.java:204)
  at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:1121)
  at org.elasticsearch.upgrades.MlTrainedModelsUpgradeIT.putAndStartDFAAndWaitForFinish(MlTrainedModelsUpgradeIT.java:200)
  at org.elasticsearch.upgrades.MlTrainedModelsUpgradeIT.createAndRunClassificationJob(MlTrainedModelsUpgradeIT.java:191)
  at org.elasticsearch.upgrades.MlTrainedModelsUpgradeIT.testTrainedModelInference(MlTrainedModelsUpgradeIT.java:65)
  at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(NativeMethodAccessorImpl.java:-2)
  at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
  at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:568)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:833)

```

--- a/x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/MlTrainedModelsUpgradeIT.java
+++ b/x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/MlTrainedModelsUpgradeIT.java
@@ -62,7 +62,7 @@ public class MlTrainedModelsUpgradeIT extends AbstractUpgradeTestCase {
             case OLD -> {
                 createIndexWithName(INDEX_NAME);
                 indexData(INDEX_NAME, 1000);
-                createAndRunClassificationJob();
+                createAndRunClassificationJob("classification-upgrade-job");
                 createAndRunRegressionJob();
                 List<String> oldModels = getTrainedModels();
                 createPipelines(oldModels);
@@ -172,7 +172,7 @@ public class MlTrainedModelsUpgradeIT extends AbstractUpgradeTestCase {
         putAndStartDFAAndWaitForFinish(config, "regression");
     }
 
-    void createAndRunClassificationJob() throws Exception {
+    void createAndRunClassificationJob(String jobName) throws Exception {
         String config = Strings.format("""
             {
               "source": {
@@ -188,7 +188,7 @@ public class MlTrainedModelsUpgradeIT extends AbstractUpgradeTestCase {
               },
               "model_memory_limit": "18mb"
             }""", INDEX_NAME, KEYWORD_FIELD);
-        putAndStartDFAAndWaitForFinish(config, "classification");
+        putAndStartDFAAndWaitForFinish(config, jobName);
     }
 
     @SuppressWarnings("unchecked")
@@ -202,7 +202,7 @@ public class MlTrainedModelsUpgradeIT extends AbstractUpgradeTestCase {
                 client().performRequest(new Request("GET", "_ml/data_frame/analytics/" + id + "/_stats"))
             ).get("data_frame_analytics")).get(0);
             assertThat(state.get("state"), equalTo("stopped"));
-        }, 1, TimeUnit.MINUTES);
+        }, 2, TimeUnit.MINUTES);
     }
 
     void createPipeline(String id, String modelType, String modelId) throws Exception {

